### DISCUSSION TITLE
Add more repository management options in user/organization settings

### DISCUSSION THREAD

**Discussion by ferferga** [community/community#4125] [Other Feature Feedback, Questions, & Ideas]
Hello

In https://github.com/settings/repositories there are options to leave repositories where you're a collaborator or owner and see the size of each repository. It would be great if this settings section had all the possible repository options such as **Delete**, **Rename**, **Transfer** or **Archive**. Right now it's possible to only **Leave**, which is confusing as well, as that option is not present at all in the repo's own settings section. 
Adding all those options to this page would ease the management for users and organizations with multiple repositories, as currently we need to do it one repository at a time.

Additionally, it would be great if to this page we get an option added to request a [repository purge](https://docs.github.com/en/github/authenticating-to-github/keeping-your-account-and-data-secure/removing-sensitive-data-from-a-repository#using-filter-branch) (step 8) so we, users, don't need to contact GitHub support for those situations, not only when we want to delete data, but also when we make a huge rebase and force push and our commit history in our profile and repo are still linking to the old and stale commits.

Thank you very much in advance.

**Comment by tallguyjenks** [comment]
also a great addition to this is kind of a "will and testament" for the code base. Upon death of maintainer, if that maintainer was the sole maintainer of the project and no one has rights to it, it would be nice to have an option to tap someone as next in line.

> Upon my death, this repository and all rights to it will be transferred henceforth to John Doe
